css/layout.css

```diff
@@ -0,0 +1,66 @@
+.container {
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+.row {
+  display: flex;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-fourth {
+  flex-basis: 25%;
+}
+
+.column-half {
+  flex-basis: 50%;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.flex-row {
+  display: flex;
+}
+
+@media screen and (width <= 375px) {
+  .row {
+    flex-direction: column;
+  }
```

Comment:
I would not make this switch from row to column on smaller screen. Is this for your movie cards? If so, why not just let the container wrap and let the items go to different lines?